docs/assets/js/main.js

```diff
@@ -75,3 +66,10 @@ if (window.tocbot) {
         },
     });
 }
+
+document.getElementById('header-button').addEventListener('click', toggleHeaderMenu);
```

Comment:
This is probably OK, but I think only because of the `defer` on the original script tag. Would you mind adding these event listeners only once the page-ready event (eg. `DOMContentLoaded`) has fired just to be extra careful?

Reply:
Sure, I moved those listeners (and also the `tocbot.init()` function) to the `DOMContentLoaded` event